src/components/FAQ.js

```diff
@@ -33,28 +33,21 @@ function FAQ() {
             {questions.map((question) => (
```

Comment:
You could destructure questions but I'm only writing this, because I didnt find anything else :))